### PR TITLE
Add simple timer

### PR DESCRIPTION
**Pull request by fuzzytern**
Adds a basic implementation for the simple timer specified in issue #2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,20 +5,32 @@
...
+/**
+ * Methods 
+ */
+const runInterval = function (duration, interval = 60000) {
+  console.log(`${duration}mn left`);
+  if (duration === 0) {
+    console.log("Time's up");
+  } else {
+    // calls a copy of runInterval recursively with the updated parameters
+    setTimeout(runInterval.bind(null, duration - 1, interval),
```
Comment (hayko1995) [inline index.js]: Why you use bind? At last yo have   many functions and that not good for ram. think use recursion.

**Comment by fuzzytern** [comment]
I decided to use `setTimeout` with params instead of `.bind` as it makes the syntax more expressive (less characters and parenthesis). I am still unsure whether this recursive approach is preferable over using `setInterval` imperatively in terms of memory use and performance.

Until I research it let me know if you have an opinion on it, or any other comments.

**Comment by fuzzytern** [comment]
@armfashion @paerallax @ArpenTadevosyan @ArmenSan

Any reviews or comments? If you can't think of any valuable feedback but are confident that this code can be merged safely, please approve this pull request (this won't merge it).

**Comment by Mikazakaryan** [comment]
i think
if(duration <= 0) {
console.log("Time's up");
}
will be better
give duration 0.1 sec end you will see ;)

**Pull request merged**